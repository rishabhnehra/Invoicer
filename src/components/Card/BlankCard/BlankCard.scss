.BlankCard {
    display: inline-block;
    border-radius: 10px;
    padding: 16px 24px;
    background-color: #E2F3F5;
    box-shadow: 0px 0px 6px 3px rgba(0,0,0,0.1);
}