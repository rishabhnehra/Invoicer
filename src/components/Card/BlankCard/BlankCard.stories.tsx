import {Story, Meta} from '@storybook/react/types-6-0';

import {BlankCard, BlankCardProps} from './BlankCard';
import { TextInput } from '../../index'

export default {
    title: 'Components/Card',
    component: BlankCard
} as Meta;

const Template: Story<BlankCardProps> = (args) => <BlankCard {...args} />

export const Blank = Template.bind({});
Blank.args = {
    children: <>
        <div>
            <TextInput placeholder="Email" type="email" onClick={() => null} />
        </div>
    </>
}