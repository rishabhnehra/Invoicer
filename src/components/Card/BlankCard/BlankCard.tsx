import './BlankCard.scss'

export type BlankCardProps = {
    className: any;
    children: any;
}

export const BlankCard: React.FC<BlankCardProps> = ({
    className,
    children
}) => {
    return <div className={`BlankCard ${className}`}>
        {children}
    </div>
}