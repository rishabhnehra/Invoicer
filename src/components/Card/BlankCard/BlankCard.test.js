import renderer from 'react-test-renderer';
import { BlankCard } from './BlankCard';

describe("<BlankCard />", () => {
    it("snapshot check", () => {
        const tree = renderer.create(<BlankCard>
            <h1>Any</h1>
            <p>element</p>
        </BlankCard>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});