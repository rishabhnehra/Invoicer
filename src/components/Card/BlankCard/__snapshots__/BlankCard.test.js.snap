// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlankCard /> snapshot check 1`] = `
<div
  className="BlankCard undefined"
>
  <h1>
    Any
  </h1>
  <p>
    element
  </p>
</div>
`;
